src/Approximations/overapproximate_expmap.jl

```diff
@@ -156,6 +156,61 @@ function load_intervalmatrices_overapproximation_expmap()
             res = overapproximate(E * Z, Zonotope)
             return res
         end
+
+        """
+            overapproximate(expA::MatrixZonotopeExp{N,T}, ::Type{<:MatrixZonotope},
+                                 k::Int=2) where {N,T<:AbstractMatrixZonotope{N}}
+        
+        Overapproximate the matrix zonotope exponential ``exp(\\mathacal{A})``
+
+        ### Input
+
+        - `expA` -- A `MatrixZonotopeExp`
+        - `MatrixZonotope` -- target type
+        - `k` -- (default: `2`) the order of the taylor expansion
+
+        ### Output 
+        
+        A matrix zonotope overapproximating the matrix zonotope exponential
+
+        ### Algorithm
+        
+        The algorithm follows [AlthoffKS11](@citet). 
```

Comment:
If there is no pointer to an equation/proposition for the first part (expansion), consider moving the reference to the end ("Proposition 4.1").